packages/cxl-institute/scss/global.scss

```diff
@@ -0,0 +1,299 @@
+@import "@conversionxl/cxl-ui/scss/utils/mixins";
+@import "@conversionxl/cxl-ui/scss/cxl-theme";
+
+body,
+html {
+  overflow: hidden;
+}
+
+body {
+  background: var(--lumo-tint-20pct);
+
+  @include respond-below(small) {
+    background: var(--lumo-base-color);
+  }
+}
+
+.menu {
+  &--institute-icon-nav {
+    background: #ebebeb;
+    border-right: 3px solid var(--lumo-primary-color);
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    width: 64px;
+  }
+
+  &__items {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__item {
+    align-items: center;
+    display: flex;
+    height: 64px;
+    position: relative;
+    width: 100%;
+
+    a {
+      align-items: center;
+      display: flex;
+      height: 100%;
+      width: 100%;
+
+      &:hover {
+        background: var(--lumo-primary-color);
+      }
+    }
+
+    &.has-children:first-child {
+      height: 100%;
+    }
+    &.has-children:last-child {
+      height: 64px;
+    }
+  }
+
+  &__link {
+    display: none;
+  }
+
+  &__sub-menu {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+  }
+}
+
+.icon-nav {
```

Comment:
I don't get it, old styles are still sticking around? Is this the same as `.menu--institute-icon-nav`?

Reply:
Yes, I have kept the old one with stories extra layout until icon nav with WP markup gets completed.

Reply:
@chanar